app/docs/markdown/getting-started-with-dockstore.md

```diff
@@ -230,6 +230,64 @@ The `dockstore tool manual_publish` command can be used to manually register a t
 
     ------------------
 
+## Additional Information on Build Modes
+
+#### Fully-Automated
+
+How to create it:
+
+You create it by using the "Refresh All Tools" button.  This will scan through your Quay.io repositories and automatically register the tool on Dockstore.
+
+Requirements:
+
+- Using a Quay.io registry with Quay.io linked to Dockstore.
+
+Benefits:
+
+This build mode automatically adds versions to your tool with Quay tags and Git reference using the Quay.io build trigger.
+
+Limitations:
+
+- Unable to deregister tool
+- Unable to manually add versions
+- Cannot uniquely name the tool
+
+When to use:  
+
+Recommended when you are using a Quay.io registry, want a quick and easy way to register the tool, and want to avoid manually adding new versions to the tool.  Generally recommended for most tools.
+
+#### Manual
+
+How to create it:
+
+- Click Add Tool, use Quay.io registry, and change the tool name from default.
```

Comment:
You can use any registry with manual. 
A tool name that is changed from the default is only necessary if there is a name clash